ime/app/src/main/java/com/anysoftkeyboard/ime/AnySoftKeyboardBase.java

```diff
@@ -329,4 +351,62 @@ public void onCancel() {
         // the user released their finger outside of any key... okay. I have nothing to do about
         // that.
     }
+
+    @RequiresApi(Build.VERSION_CODES.R)
+    @Nullable
+    @Override
+    public InlineSuggestionsRequest onCreateInlineSuggestionsRequest(@NonNull Bundle uiExtras) {
+        Size smallestSize = new Size(0, 0);
+        Size biggestSize = new Size(Integer.MAX_VALUE, Integer.MAX_VALUE);
+
+        UiVersions.StylesBuilder stylesBuilder = UiVersions.newStylesBuilder();
+
+        InlineSuggestionUi.Style style = InlineSuggestionUi.newStyleBuilder().build();
+        stylesBuilder.addStyle(style);
+
+        Bundle stylesBundle = stylesBuilder.build();
+
+        InlinePresentationSpec spec = new InlinePresentationSpec.Builder(smallestSize, biggestSize).setStyle(stylesBundle).build();
+
+        List<InlinePresentationSpec> specList = new ArrayList<>();
+        specList.add(spec);
+
+        InlineSuggestionsRequest.Builder builder = new InlineSuggestionsRequest.Builder(specList);
+
+        return builder.setMaxSuggestionCount(InlineSuggestionsRequest.SUGGESTION_COUNT_UNLIMITED).build();
+    }
+
+    @RequiresApi(Build.VERSION_CODES.R)
+    @Override
+    public boolean onInlineSuggestionsResponse(@NonNull InlineSuggestionsResponse response) {
+        List<InlineSuggestion> inlineSuggestions = response.getInlineSuggestions();
+
+        if (mInputViewContainer == null) {
```

Comment:
Do not create the view manually. Let the OS do that.

I suggest that if the method is called before the view was created then don't do anything. Is that possible?

Reply:
I can attempt that. In my mind it would make sense that the view is already created, but I can throw that code into the if check there and turn it off in case the view doesn't exist. I could also potentially cache the suggestions until the view is available, but that would open a whole other can of worms

Reply:
removed the check and all works well! thanks for the note!